src/hats/io/parquet_metadata.py

```diff
@@ -321,20 +348,24 @@ def per_pixel_statistics(
         exclude_hats_columns=exclude_hats_columns,
         exclude_columns=exclude_columns,
         include_columns=include_columns,
+        only_numeric_columns=only_numeric_columns,
     )
     if not good_column_indexes:
         return pd.DataFrame()
 
     all_stats = ["min_value", "max_value", "null_count", "row_count"]
+    int_stats = ["null_count", "row_count"]
```

Comment:
So if I understand, `null_count` and `row_count` will get coerced to int, which is good, but `min_value` and `max_value` can't, because the meaning depends on the type.  Which makes sense!  But then I wonder about how the stringized values get into there in the first place and whether it can be prevented there.